### ISSUE TITLE
JMP 1.11.1 removes trailing slash which makes it unable to connect to my Jellyfin server

### ISSUE THREAD

**Issue by fredguile** [jellyfin/jellyfin-media-player#754]
Four considerations:
 - Please do not open bug reports to ask questions. Use the Discussions feature instead.
 - Please make sure the issue only pertains to Jellyfin Media Player. If it also occurs in the web client, send the issue to jellyfin-web instead.
 - Please make sure that your issue is not being caused by errors in custom CSS or note that you are using custom CSS.
     - Notably, there have been instances of custom CSS breaking TV mode.
     - You can disable custom CSS under Display in the user settings.
 - Please provide logs. You can drag the log file into the issue to attach it.
     - Windows: `%LOCALAPPDATA%\JellyfinMediaPlayer\logs`
     - Linux: `~/.local/share/jellyfinmediaplayer/logs/`
     - Linux (Flatpak): `~/.var/app/com.github.iwalton3.jellyfin-media-player/data/jellyfinmediaplayer/logs/`
     - macOS: `~/Library/Logs/Jellyfin Media Player/`

**Describe the bug**
On my shared Jellyfin server, everything is secured via basic auth with exception of that URL "/username/jellyfin/" and sadly I don't have any way to change that (hint: it's a well-known service used by many).

As since this [commit](https://github.com/jellyfin/jellyfin-media-player/commit/3f31831a7efc6e00ced7ac67410abdaa8b68faa1), JMP 1.11.1 will strip out the trailing slash, it makes it unable to connect, as basic auth isn't supported AFAIK.

I would like to open a tiny fix PR touching "find-webclient.js" in order to unblock this.

**To Reproduce**
You need to assume that your Jellyfin server is secured with basic auth on pretty much every URL except "/jellyfin/*" (that trailing slash is of a utmost importance).

1. Just log-in as usual, the login process wouldn't succeed as failing on a 401 auth required as seen in the dev tools

**Expected behavior**
We could just not slash that trailing slash, this would solve the issue. 

**Desktop (please complete the following information):**
 - not OS-specific
 - I've been testing a fix on version 1.11.1 and linux, I can open a fix PR shortly

**Additional context**
That bug seems low priority, but it probably affects many JMP users utilizing a well-known app sharing provider.

**Comment by andrewrabert** [comment]
Fixed in v2.0
